.devcontainer/postCreate.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Old school 😎

Reply:
I like to think it's more portable even in modern times! :)